salt/states/restconf.py

```diff
@@ -0,0 +1,188 @@
+"""
+State module for restconf Proxy minions
+
+:codeauthor: Jamie (Bear) Murphy <jamiemurphyit@gmail.com>
+:maturity:   new
+:platform:   any
+
+About
+=====
+This state module was designed to manage restconf states.
+This module relies on the restconf proxy module to interface with the devices.
+"""
+
+
+import json  # noqa: F401
+
+# from salt.utils.odict import OrderedDict  # noqa: F401
+
+try:
+    HAS_DEEPDIFF = True
+    from deepdiff import DeepDiff
+except ImportError:
+    HAS_DEEPDIFF = False
+
+
+def __virtual__():
+    if not HAS_DEEPDIFF:
+        return (
+            False,
+            "Missing dependency: The restconf states method requires the 'deepdiff' Python module.",
+        )
+    if "restconf.set_data" in __salt__:  # noqa: F821
```

Comment:
**suggestion**: I just remembered that we prefer the names rather than codes, e.g. `undefined-variable-name` or whatever F821 maps to.

Reply:
will remove these. was using these for flake8 internally

Reply:
removed all these as they were from flake8 testing internally